package.json

```diff
@@ -30,7 +30,7 @@
     "prepare": "husky install client/.husky",
     "lint": "yarn eslint",
     "prettier": "yarn pretty-quick",
-    "predeploy": "yarn build",
+    "predeploy": "yarn run build",
```

Comment:
The run command in Yarn is optional, for NPM is necessary: 
```bash
yarn build //works
yarn run build //works
npm run build //works
npm build //won't work 
```
[Here](https://stackoverflow.com/questions/40619269/what-is-the-difference-between-yarn-run-and-npm-start) you can read more about it, is one of the reasons I rather use yarn.

Reply:
That being said, I don't think this will fix the issue. Could be wrong though